housekeeping-vacuum-tool/pom.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>housekeeping-parent</artifactId>
+    <groupId>com.hotels</groupId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>housekeeping-vacuum-tool</artifactId>
+  <name>Vacuum tool</name>
+
+  <dependencies>
+    <dependency>
```

Comment:
Where do all these dependencies come from original pom didn't have these

Reply:
All of them? I can understand the spring boot stuff.
But are there are errors when you omit these? `javax.el-api` and `javax.el` This is for SpEL parsing which we use in partitionPredicates which isn't used in the vacuum-tool.

The `circus-train-avro` also really confuses me, we shouldn't use any of that code.

Reply:
I'm not sure how the avro thing snuck in there, I will do a test run without javax stuff, I think I ran into some Spring issues at run time last time I tested without it but that was when depending on CT, so might be a different case now